### ISSUE TITLE
Checksums point to trac server

### ISSUE THREAD

**Issue by roed314** [sagemath/sage#35014]
The following four checksums are hosted on the Sage trac server as an attachment, and should probably move:

```
build/pkgs/giac/checksums.ini:upstream_url=https://trac.sagemath.org/raw-attachment/ticket/31563/giac-VERSION.tar.bz2
build/pkgs/iml/checksums.ini:upstream_url=https://trac.sagemath.org/raw-attachment/ticket/33195/iml-1.0.4p2.tar.bz2
build/pkgs/mathjax/checksums.ini:upstream_url=https://trac.sagemath.org/raw-attachment/ticket/25833/mathjax-3.2.0.tar.gz
build/pkgs/msolve/checksums.ini:upstream_url=https://trac.sagemath.org/raw-attachment/ticket/31664/msolve-VERSION.tar.gz
```

**Comment by mkoeppe** [comment]
These URLs are only important during upgrade cycles when the ticket has not yet been merged by the release manager.
The release manager uses these URLs to download the upstream tarballs and deposits them on our file server.

**Comment by roed314** [comment]
So, should we close this issue as irrelevant or update the URLs to point to the analogous files on github?  I didn't want to hide that in a giant PR (#35015) since I wasn't sure how it would affect building Sage.

**Comment by dimpase** [comment]
we can just delete these as no longer relevant, no?

**Comment by dimpase** [comment]
the giac tarballs there are definitely not needed

**Comment by dimpase** [comment]
ditto for mathjax

**Comment by roed314** [comment]
Ok, I will create a PR deleting these four lines, but I definitely want to get a signoff from someone who knows that each link is no longer needed.

**Comment by dimpase** [comment]
sure. Also msolve is an active project, and iml is maintained. So these tarballs were added on trac for convenience of automatic (or not) testing, and are now on Sage mirrors.

**Comment by dimpase** [comment]
fixed by #35023
